changelog.md

```diff
@@ -1,6 +1,8 @@
 ## Changelog
 ### Unreleased
 - Drop support for Python 2
+- cxroots now logs rootfinding progress which can be accessed using the standard library [logging module](https://docs.python.org/3/library/logging.html). See the [documentation](https://rparini.github.io/cxroots/logging.html) for examples.
+- The `verbose` argument has been removed from some functions and for the `Contour.roots` method or `find_roots` function it will now create a progress bar, using [rich](https://github.com/Textualize/rich), rather than printing debugging information to the console.
```

Comment:
```suggestion
- The `verbose` argument has been removed from some functions and for the `Contour.roots` method or `find_roots` function it will now create a progress bar, using [rich](https://github.com/Textualize/rich), rather than printing debugging information to the console.
```
```suggestion
- The `verbose` argument has been removed from some functions. It still exists for the `Contour.roots` method and `find_roots` function but it will now create a progress bar, using [rich](https://github.com/Textualize/rich), rather than printing debugging information to the console.
```